specification/dimensions/commitmentutilization.md

```diff
@@ -0,0 +1,37 @@
+# Commitment Utilization
```

Comment:
I am personally not in favor of the use of the term "Utilization" in this fashion.  To my mind, the term "Utilization" implies a crucial FinOps KPI that, as we all know, is a percentage measure that returns the result of Used / (Used + Not Used).  However, here, it is being used to describe a attribute that one could use in order to derive that calculation.  This introduces a semantic conflict later, if and when the FOCUS spec add "Utilization" to the list of Metrics (which I hope it does!).

I feel that the term "Utilized" or "Utilized Flag" would not only help delineate from that, but it would also be more self-documenting and imply that this is an attribute with two (and only two) possible values.

I also prefer the members of a flag attribute to follow the same pattern as the name of the attribute.  To that end, I propose the use of "Utilized" and "Not Utilized" instead of "Used" and "Not Used".

Reply:
This is the primary thing I was expecting to hear from people on this column 🙂 I love that "Utilization" binds it to the KPI, but it is a good point that there could possibly be confusion around the column vs. KPI.

A few other thoughts on names:
1. `CommitmentStatus` was our original name. We thought about "Status" for coverage calculation, but that would require separate columns per commitment type, so we scoped it out.
2. `CommitmentUtilized` sounds like a boolean (true/false). I'm personally not a huge fan of that word given it's not something we use often in English, but it is at least aligned to the KPI.
3. `CommitmentUsage` is another option I suppose. "Usage" is a more common term than "Utilization".

"UtilizationType" feels unnecessarily long.

I'll keep this open to get more feedback.

Reply:
I agree CommitmentUtilization refers more to a Commitment KPI, and my preference goes to 3. CommitmentUsage.

Another possibility may be "UnusedCommitment" with values "Yes" and "No".

Reply:
Added your suggestion above so they're all together. 
Tracking 2 votes against CommitmentUtilization, so that sounds good.
What do others think? Should we do a poll or can we get consensus here?

Reply:
Maybe `CommittmentUsed` or `CommitmentUtilized`?

Reply:
Vote here: https://finopsfoundation.slack.com/archives/C053CNCRVCL/p1698191641650719